ui-kit/src/main/kotlin/org/p2p/uikit/atoms/UiKitLeftAmountView.kt

```diff
@@ -0,0 +1,61 @@
+package org.p2p.uikit.atoms
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.annotation.DrawableRes
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.ContextCompat
+import org.p2p.uikit.R
+import org.p2p.uikit.databinding.WidgetLeftAmountViewBinding
+import org.p2p.uikit.utils.inflateViewBinding
+import org.p2p.uikit.utils.withImageOrGone
+import org.p2p.uikit.utils.withTextOrGone
+
+class UiKitLeftAmountView @JvmOverloads constructor(
```

Comment:
Let's split it of `UiKitStartAmountView` and `UiKitEndAmountView`

The current name is confusing a little bit, because the developer can assume that it's about some remaining value, not about the position

Reply:
But in general, these two views are absolutely different and perhaps we shouldn't make their names similar dividing to start/end or left/right. 

These two components have different text views and images so let's think about more informative names

Reply:
Yes, it's way better! I was confused about naming this guys